openapi3/content.go

```diff
@@ -12,6 +12,32 @@ func NewContent() Content {
 	return make(map[string]*MediaType, 4)
 }
 
+func NewContentWithSchema(schema *Schema, consumes []string) Content {
```

Comment:
```suggestion
func NewContentWithSchema(schema *Schema, consumes ...string) Content {
```

Reply:
Why is this needed, will this work?

Reply:
The idea is to not have to pass a `consumes` such that `len(consumes) == 0` as a second argument to trigger that empty consumes argument behaviour.

i.e. this way `NewContentWithSchema(schema, []string{}) == NewContentWithSchema(schema)`.